build/ci/unit-tests.yml

```diff
@@ -29,6 +29,7 @@ jobs:
   parameters:
     name: Windows_Debug
     configuration: Debug
+    enableRichNavigation: true
```

Comment:
Should this be enableRichCodeNavigation?

Reply:
Yes, good catch! That's why the indexing step didn't run, my bad. I'll respond to the thread once it runs to let you know how much time _might_ be added to your main CI